kubevirt_scripts/unmountcd.sh

```diff
@@ -76,7 +76,8 @@ EOF
 if [[ ${NUM_VOLUMES} -ge 1 ]]; then
   oc -n ${VM_NAMESPACE} patch vm ${VM_NAME} --patch-file /tmp/${VM_NAME}.patch --type json
   if [ $? -eq 0 ]; then
-    oc -n ${VM_NAMESPACE} delete configmap ${VM_NAME}-iso-ca &> /dev/null
+    #oc -n ${VM_NAMESPACE} delete configmap ${VM_NAME}-iso-ca &> /dev/null
```

Comment:
Was this left commented out on purpose? If it's not needed, let's remove it entirely.

Reply:
@jobbler could you remove these comments so we can get this merged?

Reply:
@mvazquezc sorry, I did not see this request. I have removed the comment.